<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tracking</class>
 <widget class="QWidget" name="tracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracking</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="main_layout" stretch="1,4,4,4">
     <item>
      <layout class="QVBoxLayout" name="tracking_control_box_layout">
       <item>
        <widget class="QGroupBox" name="tracking_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Tracking</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="stop_button">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="track_button">
            <property name="text">
             <string>Track</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="max_xyz_button">
              <property name="text">
               <string>Max XYZ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="max_xy_button">
              <property name="text">
               <string>Max XY</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="max_z_button">
              <property name="text">
               <string>Max Z</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="set_tracking_parameters_button">
              <property name="text">
               <string>Configure tracking</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="filename_label">
              <property name="text">
               <string>TRK_FILENAME</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="save_button">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../../artwork/qudi_icons.qrc">
                  <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_button">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../../artwork/qudi_icons.qrc">
                  <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="load_button">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../../artwork/qudi_icons.qrc">
                  <normaloff>:/icons/icons/load.png</normaloff>:/icons/icons/load.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="previous_button">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../../artwork/qudi_icons.qrc">
                  <normaloff>:/icons/icons/previous.png</normaloff>:/icons/icons/previous.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="next_button">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../../artwork/qudi_icons.qrc">
                  <normaloff>:/icons/icons/next.png</normaloff>:/icons/icons/next.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="xy_scan_layout" stretch="0">
       <item>
        <widget class="PlotWidget" name="tracking_points_plot" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="axis_scan_layout">
       <item>
        <widget class="PlotWidget" name="fast_scan_plot" native="true"/>
       </item>
       <item>
        <widget class="PlotWidget" name="slow_scan_plot" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="z_scan_layout">
       <item>
        <widget class="PlotWidget" name="z_scan_plot" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../artwork/qudi_icons.qrc"/>
 </resources>
 <connections/>
</ui>
